test/e2e/adminapi_redeployvm.go

```diff
@@ -80,7 +82,7 @@ var _ = Describe("[Admin API] VM redeploy action", func() {
 func getNodeUptime(g Gomega, ctx context.Context, node string) (time.Time, error) {
 	// container kernel = node kernel = `uptime` in a Pod reflects the Node as well
 	namespace := "default"
-	name := node
+	name := fmt.Sprintf("%s-uptime-%s", node, utilrand.String(5))
```

Comment:
Instead of using a rand function, can we leverage `GinkgoParallelProcess()` to make it unique but deterministic? https://onsi.github.io/ginkgo/#patterns-for-testing-against-singletons

Reply:
Updated to use `GinkgoParallelProcess()`